src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -40,4 +40,36 @@ public void end(boolean interrupted) {}
   public boolean isFinished() {
     return false;
   }
-}
+
+  //take many botton's
+  private int movewristp() {
```

Comment:
nitpick: Follow camel case naming.
```suggestion
  private int moveWristPos() {
```